### PR TITLE
fix(hubble): fix the way to generate executor name of the func hub pull

### PR DESCRIPTION
**Pull request by floralatin**
Fix the way to generate executor name of the func hub pull

Close: https://github.com/jina-ai/hubble/issues/475

### CODE REVIEW

**Review by mapleeit — CHANGES_REQUESTED** [review]
I think the change here is invalid, have you tested it?

**Comment by floralatin** [comment]
> I think the change here is invalid, have you tested it?

https://files.slack.com/files-pri/T0189QW56UE-F03N6QCG2S3/image.png

**Review by mapleeit — APPROVED** [review]

File: jina/hubble/hubio.py [hunk 1]
```diff
@@ -787,7 +787,7 @@ def pull(self) -> str:
                     force=need_pull,
                 )
 
-                presented_id = getattr(executor, 'name', executor.uuid)
+                presented_id = getattr(executor, 'name') if getattr(executor, 'name') is not None else executor.uuid
```
Comment (mapleeit) [inline jina/hubble/hubio.py:790]: Does it make sense if we just use `executor.name` instead of `getattr(executor, 'name)`

Comment (floralatin) [reply jina/hubble/hubio.py:790]: yes

**Review by mapleeit — APPROVED** [review]

**Review by JoanFM — CHANGES_REQUESTED** [review]
Please add a test covering this

**Comment by floralatin** [comment]
> Please add a test covering this

yes, thanks

**Review by mapleeit — CHANGES_REQUESTED** [review]
The test doesn't make sense, it doesn't cover https://github.com/jina-ai/hubble/issues/475. We need to make sure assert the display name.

**Comment by floralatin** [comment]
> The test doesn't make sense, it doesn't cover [jina-ai/hubble#475](https://github.com/jina-ai/hubble/issues/475). We need to make sure assert the display name.

 let me think and  fix it

File: tests/unit/hubble/test_hubio.py [hunk 2]
```diff
@@ -375,8 +375,8 @@ def iter_content(self, buffer=32 * 1024):
     def status_code(self):
         return self.response_code
 
-
-def test_pull(test_envs, mocker, monkeypatch):
+@pytest.mark.parametrize('executer_name', ['alias_dummy',None])
```
Comment (mapleeit) [inline tests/unit/hubble/test_hubio.py:378]:
```suggestion
@pytest.mark.parametrize('executer_name', ['alias_dummy', None])
```

**Review by mapleeit — APPROVED** [review]

**Review by JoanFM — APPROVED** [review]

**Pull request merged**